### ISSUE TITLE
Importing css doesn't work with Sass

### ISSUE THREAD

**Issue by derochedesign** [chintan9/plyr-react#997]
Not sure if this is an issue with plyr itself or with plyr-react. The styles in the plyr repo seem much different than those here (they use scss, for starters).

**Describe the bug**
When importing `import "plyr-react/plyr.css";` no styles are imported. When copy/pasting the css rules into my own styles, our sass loader finds an error with this specific rule: `margin-right: calc(-7px - -2);` (which is inside of `.plyr__menu__container .plyr__menu__value`).

When I switch this to `margin-right: calc(-7px - -2px);` the styles load properly. (I'm not really even sure what this calc is trying to do)

**To Reproduce**
Bring the plyr styles into a project with sass loader.

**Desktop:**

- OS: macOS Monterey
- Browser: Firefox 107
Using plyr-react v5.1.0

**Comment by chintan9** [comment]
@realamirhe  I will do it

**Comment by derochedesign** [comment]
Hey all, just for some guidance, is this a fix we could expect in the next week or so? Or is it more like a month away? No issue either way, just looking to plan my work around it.

**Comment by realamirhe** [comment]
hey @derochedesign, in case @chintan9 miss your message, I guess it won't take more than a week. he occasionally publishes the latest version changes within his weekend.

**Comment by chintan9** [comment]
@realamirhe I published new version 5.1.1 .
Please test it @derochedesign

**Comment by realamirhe** [comment]
These two aren't updated in the master branch for me

https://github.com/chintan9/plyr-react/blob/0e6459f0b3c1da51f26d9a262969b110b7da1201/package.json#L64

https://github.com/chintan9/plyr-react/blob/0e6459f0b3c1da51f26d9a262969b110b7da1201/package.json#L156

**Comment by chintan9** [comment]
> These two aren't updated in the master branch for me
> 
> https://github.com/chintan9/plyr-react/blob/0e6459f0b3c1da51f26d9a262969b110b7da1201/package.json#L64
> 
> https://github.com/chintan9/plyr-react/blob/0e6459f0b3c1da51f26d9a262969b110b7da1201/package.json#L156

I have fixed on v5.1.2

https://cdn.jsdelivr.net/npm/plyr-react@5.1.2/package.json

**Comment by derochedesign** [comment]
The sass loader error is gone now 👍 
Looks like the css as side effect issue is also affecting me so i'll wait on that to be merged (https://github.com/chintan9/plyr-react/pull/963). Thanks for the quick work!

**Comment by realamirhe** [comment]
Hey @derochedesign

could you explain how CSS side effect affect your project, also it would be appreciated to comment on the following [comment](https://github.com/chintan9/plyr-react/pull/963#issuecomment-1334882651)

**Comment by derochedesign** [comment]
> Hey @derochedesign
> 
> could you explain how CSS side effect affect your project, also it would be appreciated to comment on the following [comment](https://github.com/chintan9/plyr-react/pull/963#issuecomment-1334882651)

Sorry for the delay; I don't actually know why, I just saw that my issues were the same as described in that issue and so I tried setting css as a side effect and it worked after that. I don't know how or why, but it just worked.

**Comment by realamirhe** [comment]
Try to make another issue for css side effect @derochedesign, please 😊
